examples/bri-3/package.json

```diff
@@ -57,6 +57,7 @@
     "@nestjs/cli": "^9.0.0",
     "@nestjs/schematics": "^9.0.0",
     "@nestjs/testing": "^9.0.0",
+    "@types/bcrypt": "^5.0.0",
```

Comment:
Still have the types here for bcrypt, but lets just take it out in next PR for sake of getting this through.